docs/02-install.md

```diff
@@ -144,6 +151,92 @@ file in your installation artifacts directory and simply re-export it:
 
     export KUBECONFIG=/path/to/something/auth/kubeconfig
 
+# Advanced Installations
+While the OpenShift 4 installer's purpose in life is to streamline operations
+in order to guarantee success, there are a few options that you can adjust by
+using a configuration file. Namely, you can change the default number of
```

Comment:
> `Namely`

This cannot be source of all the configurables available in `install-config`. `/s/Namely/For example,`

Reply:
This is a nitpick on the English language :) I did a quick look at Merriam Webster and didn't see any reference to quantity when defining how _namely_ is used. But I'll use your wording instead just in case anyone gets spicy.